tests/x11/updates_kde.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub pre_run_hook() {
+    # Turn off screensaver
+    x11_start_program("kcmshell5 screenlocker");
+    send_key("alt-l");
+    send_key("alt-o");
+}
+
+sub run() {
+    my @updates_installed_tags = qw/updates_none updates_available/;
+
+    if (check_screen("updates_available-tray")) {
+        assert_and_click("updates_available-tray");
+
+        for (1 .. 2) {
```

Comment:
Thank you for the explanation. If I ask "why…" then others can come to the same question when reading the code but then they don't have your github comment available. I recommend to make that obvious from the code.